src/UI/App.hs

```diff
@@ -147,8 +148,9 @@ initialState conf =
          (E.editor ManageFileBrowserSearchPath Nothing path)
     mailboxes = view (confComposeView . cvIdentities) conf
     s = AppState conf mi mv (initialCompose mailboxes) Nothing viewsettings fb
-  in
-    applySearch s
+  in do
+    t <- zonedTimeToUTC <$> getZonedTime
```

Comment:
so, we only set the date once?  Shouldn't we update it from the current system time each time we run a search (or alternatively, set a timer to do it?)

Reply:
Yeah actually you have a point. I had in mind that you eventually close the application and start it again and being off by a few days wouldn't hurt. But perhaps using a timer is the better choice.

Reply:
Do the simple thing first IMO - update the time each time we do a new notmuch search.